mantidproject/mantid
# Adding tf asymmetry mode to Muon Analysis 2

**Description of work.**

This adds TF asymmetry mode to the new Muon analysis GUI. It also updates the CalculateMuonAsymmetry algorithm slightly so that what the algorithm puts into the ADS can be controlled. 

**To test:**
  * In the new muon analysis gui try some tf asymmetry mode workflows and check that they work as expected.
  * Run the script from the issue and check it still produces the same results and the workspaces put into the ADS have the same names as previously.



Fixes #25763  


*This does not require release notes* because this is a change to a new GUI.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
